service/health/searcher/service_test.go

```diff
@@ -0,0 +1,67 @@
+package searcher
+
+import (
+	"context"
+	"testing"
+
+	apiextensionsfake "github.com/giantswarm/apiextensions/pkg/clientset/versioned/fake"
+	"github.com/giantswarm/health-service/flag"
+	"github.com/spf13/viper"
+
+	"github.com/giantswarm/micrologger/microloggertest"
+)
+
+func Test_Service(t *testing.T) {
+	testCases := []struct {
+		RequestFactory  func() Request
+		ExpectedFactory func() Response
+		ErrorMatcher    func(error) bool
+	}{
+		// Case 1. Having an empty request given there should be one master with
```

Comment:
Same here, we usually have the `name` field and so on to make it easier to handle these tests.

Reply:
This should be fixed in my latest commit too.